packages/frontend/editor-ui/src/stores/settings.store.ts

```diff
@@ -45,13 +45,14 @@ export const useSettingsStore = defineStore(STORES.SETTINGS, () => {
 			enabled: false,
 		},
 	});
-	const mfa = ref({ enabled: false });
+	const mfa = ref({ enabled: false, enforced: false });
 	const folders = ref({ enabled: false });
 
 	const saveDataErrorExecution = ref<WorkflowSettings.SaveDataExecution>('all');
 	const saveDataSuccessExecution = ref<WorkflowSettings.SaveDataExecution>('all');
 	const saveManualExecutions = ref(false);
 	const saveDataProgressExecution = ref(false);
+	const isMFAEnforced = ref(false);
```

Comment:
Not sure to see the difference with line 48 new `enforced` field of the `mfa` ref